styles/tree-view.less

```diff
@@ -45,10 +45,20 @@
   padding-right: @component-padding;
   position: relative;
 
-  // This fixes #110, see that issue for more details
-  .entry:before {
-    content: '';
-    position: absolute;
+  .entry {
+    .name {
+      pointer-events: none;
+    }
+
+    &.dragenter {
+      background-color: rgba(255, 255, 255, 0.1);
```

Comment:
It would be great if this used something from `ui-variables` so that this color can be explicitly changed by themes